www/includes/easyparliament/templates/html/mp/profile.php

```diff
@@ -416,6 +416,30 @@
                             <p>This data was produced by TheyWorkForYou from a variety
                                 of sources. Voting information from
                                 <a href="http://www.publicwhip.org.uk/mp.php?id=uk.org.publicwhip/member/<?= $member_id ?>&amp;showall=yes">Public Whip</a>.</p>
+
+                          <?php if($image && $image['exists']) {
+                              if(isset($data['photo_attribution_text'])) { ?>
```

Comment:
Does this mean there are pages (e.g. voting record) where a photo is displayed without attribution? Could this be bottom of all pages?

Reply:
I’ve now added it to `_vote_footer.php` as well, so it makes its way onto the other `/mp` subpages. 👍